src/toil/batchSystems/lsf.py

```diff
@@ -298,3 +298,10 @@ def obtainSystemConstants(self):
         if self.maxCPU is 0 or self.maxMEM is 0:
                 RuntimeError("lshosts returns null ncpus or maxmem info")
         logger.info("Got the maxCPU: %s" % (self.maxMEM))
+
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
```

Comment:
No need to duplicate the overridden method's docstring (unless you promise to keep these copies consistent from now until all eternity).